applications/pusher/src/pusher_listener.erl

```diff
@@ -68,7 +68,8 @@ handle_push(JObj, _Props) ->
     TokenType = kz_json:get_value(<<"Token-Type">>, JObj),
     Module = kz_term:to_atom(<<"pm_",TokenType/binary>> , 'true'),
     lager:debug("pushing for token ~s(~s) to module ~s", [Token, TokenType, Module]),
-    gen_server:cast(Module, {'push', JObj}).
+    JObj1 = kz_json:insert_value(<<"timestamp">>, kz_time:current_tstamp(), JObj),
```

Comment:
Did they want `timestamp` or `ts`?

Reply:
*"ts" field (or "timestamp" or whatever you prefer)*

Reply:
ok!